.github/workflows/ci.yml

```diff
@@ -113,7 +114,15 @@ jobs:
           PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER: ${{ matrix.PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER }}
           SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.SYMFONY_DEPRECATIONS_HELPER }}
         run: |
-          vendor/bin/phpunit
+          vendor/bin/phpunit ${{ matrix.phpunit-coverage }}
+
+      - name: Upload coverage results to Coveralls
+        if: matrix.phpunit-coverage != ''
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          composer require --dev php-coveralls/php-coveralls
```

Comment:
it's already in dependency, right ? no need to manually install

Reply:
yeah, copy/pasting went bad here